tests/test_x_to_parquet.py

```diff
@@ -0,0 +1,97 @@
+"""Test conversion from CSV to Parquet using top-level function."""
+
+import warnings
+from pathlib import Path
+
+import pytest
+
+import clim2parquet
+
+
+# Test all data-specific functions and main wrapper function clim_to_parquet()
+# for admin level 0 (country level)
+# Test conversion from specific data source
+def test_clim_to_parquet(tmp_path: Path) -> None:
```

Comment:
I've not been generally doing typing on the tests, though it's obviously ok. it should be possible to disable checking for the tests directory